packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -256,13 +256,20 @@ public void chooseVideoFromGallery(MethodCall methodCall, MethodChannel.Result r
   }
 
   private void launchPickVideoFromGalleryIntent() {
-    Intent pickVideoIntent =
-        new ActivityResultContracts.PickVisualMedia()
-            .createIntent(
-                activity,
-                new PickVisualMediaRequest.Builder()
-                    .setMediaType(ActivityResultContracts.PickVisualMedia.VideoOnly.INSTANCE)
-                    .build());
+    Intent pickVideoIntent;
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
I'm not sure this check is sufficient, as Android Go doesn't support photo picker until T.

https://developer.android.com/training/data-storage/shared/photopicker

This medium article has a little more information about this.

https://medium.com/tech-takeaways/android-13-photo-picker-with-the-activity-result-api-b4a74572e354

Reply:
@tarrinneal I thought because we are using Android x `ActivityResultContracts.PickVisualMedia()` that, it would handle the SDK/photo picker compatibility for us.

If the device's SDK is equal to or above 19 then the `PickerVisualMedia` should take care of any other compatibility issues, right?

Reply:
oh you're totally right, I forgot that android x was handling that check itself